package.json

```diff
@@ -34,44 +34,23 @@
         "rxjs": "^7.8.1"
     },
     "devDependencies": {
-        "@eslint/eslintrc": "^3.3.1",
-        "@eslint/js": "^9.27.0",
-        "@jsdevtools/coverage-istanbul-loader": "^3.0.5",
-        "@morgan-stanley/ts-mocking-bird": "^1.1.3",
-        "@types/node": "^24.3.0",
-        "@types/uuid": "^11.0.0",
-        "@typescript-eslint/eslint-plugin": "^8.32.1",
-        "@typescript-eslint/parser": "^8.32.1",
-        "acorn": "^8.7.0",
-        "ajv": "^8.12.0",
-        "chromedriver": "^142.0.0",
-        "circular-dependency-plugin": "^5.2.2",
-        "concurrently": "^9.0.1",
-        "copyfiles": "^2.4.1",
-        "core-js": "^3.15.2",
-        "eslint": "^9.27.0",
-        "eslint-config-prettier": "^10.0.1",
-        "eslint-plugin-import": "^2.24.2",
-        "eslint-plugin-prefer-arrow": "^1.2.3",
-        "eslint-plugin-prettier": "^5.4.0",
-        "eslint-plugin-simple-import-sort": "^12.0.0",
-        "event-hooks-webpack-plugin": "^3.0.1",
```

Comment:
removed a lot of legacy dependencies that we don't need anymore like puppeteer, chromedriver, webpack and so on